test/testUtils/ufeUtils.py

```diff
@@ -105,3 +105,26 @@ def ufeFeatureSetVersion():
     #   v0.2.20 (unreleased preview version 2).
     major = ufe.VersionInfo.getMajorVersion()
     return ufe.VersionInfo.getMinorVersion() if major == 0 else major
+
+def hasMinimumVersion(releaseVersions, unreleasedVersions):
```

Comment:
You shouldn't need this.  See
https://github.com/Autodesk/maya-usd/pull/2096/files
on the approach we've chosen: we use the Maya preview release version and the UFE preview version to determine whether code should be compiled and tests run.  Then, when the next Maya preview release is released, we clean up ALL the preview release and UFE preview release conditions we're put in, and replace them with UFE feature set version checks.

This cleans up the code one preview release at a time so that we don't leave old unreleased version checks in the files.

Reply:
In this case I till need to differentiate between 0.4.3 and 0.4.4 though, so I'll test that only intstead.